.dockerignore

```diff
@@ -0,0 +1,6 @@
+# Ignore these directories when building images. Note some may be populated anyway on composer install.
+.git
+vendor
+web/sites/default/files
+web/core/tests
+web/modules/contrib/webform/tests
```

Comment:
why is this specific directory is in here?

also, for development, how would developers develop their tests?

Reply:
Webform just happens to have a relatively large test directory, like 50mb (from memory). So I thought i'd add it here to generate discussion like this.

I don't think we should be building a container for developers to work on core or webform.